### DISCUSSION TITLE
MayaUSD Installer Executable - Run Silently

### DISCUSSION THREAD

**Discussion by brycegbrazen** [Autodesk/maya-usd#1865] [Q&A] [answered]
Is there a way to run the MayaUSD installer silently? Particularly I'm looking to run the `MayaUSD_0.13.0_Maya2022.2_Windows.exe` installer silently on Windows.

**Comment by santosg87** [comment, marked as answer]
I did some testing on my side, and found a _possible_ way to do it, although its not an very graceful way for it:

the .exe installer file doesn't have a quiet mode, you can, however, extract it (using something like 7zip, for example) and run the MayaUSD.msi file silently. using:
`cd [path to extracted installer folder]`
`.\MayaUSD.msi /quiet`

Make sure to run the terminal window with admin permissions, as the installer will not prompt for permissions, and simply fail silently. :)

hope this helps :)
